gui.py

```diff
@@ -93,7 +93,7 @@ def start_program(self):
         self.root.update_idletasks()  # Force the window to update its size
 
     def browse_files(self):
-        file_path = filedialog.askopenfilename(initialdir="./bml_examples", title="Select a File",
+        file_path = filedialog.askopenfilename(initialdir="/", title="Select a File",
```

Comment:
![image](https://github.com/user-attachments/assets/5530a305-6b86-49b5-b813-9e1c77238bd6)

Leave the default directory as `./bml_examples`. We want the end result to be a full file browser like in the image above.

Reply:
See if this is helpful: https://www.geeksforgeeks.org/file-explorer-in-python-using-tkinter/